src/classes/CRLP_Operation_SVC.cls

```diff
@@ -74,10 +74,10 @@ public class CRLP_Operation_SVC {
         // Get the Date value from the appropriate object. There should always be a date!
         // It's a LOT faster the use the actual field name to retrieve the date than to use .get(fieldName).
         // Since the Date is almost always the CloseDate field, hardcoding this performance tweak.
-        SObject dateSObject = detailSObject;
         if (rollup.dateObject == oppObjectName && dateField == 'CloseDate') {
             dateValue = oppSObject.CloseDate;
         }
+        SObject dateSObject = rollup.dateObject == 'Opportunity' ? oppSObject : detailSObject;
```

Comment:
Can we use the parameter variable "detailSObject" instead of creating a new one "dateSObject".

Reply:
So just reuse it like:
detailSObject = rollup.dateObject == 'Opportunity' ? oppSObject : detailSObject;

I don't think we can do that since detailSObject is reused later, so resetting it with a ternary may have adverse effects later on.

Reply:
I see your point. I missed we are assigning "oppSObject" as well. That is good then, thank you!